extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelRoutesBuilderClassBuildItem.java

```diff
@@ -17,20 +17,23 @@
 package org.apache.camel.quarkus.core.deployment;
 
 import io.quarkus.builder.item.MultiBuildItem;
-import io.quarkus.runtime.RuntimeValue;
 import org.apache.camel.RoutesBuilder;
+import org.jboss.jandex.DotName;
 
 /**
- * Holds the {@link RoutesBuilder} {@link RuntimeValue}.
+ * A {@link MultiBuildItem} holding class names of all {@link RoutesBuilder} implementations found via Jandex.
```

Comment:
Not very sure if `implementations found via Jandex` really make sense as any extension developer can create it programmatically.

Reply:
True, let me reword it.